jiant/trainer.py

```diff
@@ -436,6 +435,9 @@ def get_sampling_weights(
             sample_weights = 1 / task_n_train_examples
         elif weighting_method == "inverse_log_example":
             sample_weights = 1 / np.log(task_n_train_examples)
+        elif "examples-proportional-mixing" in weighting_method:
+            max_K = int(weighting_method.split("K=")[1])
```

Comment:
Heads up: this will throw a `ValueError` if a user follows the example in the `defaults.conf`:

"...`set weighting_method=examples-proportional-mixingK==104857`"